### PR TITLE
Update DualView Kotlin sample with the new design

### PR DESCRIPTION
**Pull request by bimiron**
- Changed ListFragment to include a RecyclerView of Restaurant items
- Added a ViewModel to keep the selected item in dual mode when the orientation is changed
- Added toolbar items with different states for single and dual mode
- Changed map images to include markers
- Updated tests

### CODE REVIEW

**Comment by joyl1216** [comment]
Would you please attach the screenshots for the design review? Thanks ^_^

**Comment by bimiron** [comment]
**Single Screen Mode**
Screenshot 2020-11-17 135845
**Using the map menu icon**
Screenshot 2020-11-17 135902
**Using the search menu icon**
Screenshot 2020-11-17 135932
**Selecting an item**
Screenshot 2020-11-17 135753
**Rotating the device, the item is still selected**
Screenshot 2020-11-17 140030

**Comment by joyl1216** [comment]
@Mehul1995 Can we remove the search button, so we can get rid of the mock message? Since the Map/List icon is working, but the search button is not, it will be confused if we put them both together.

**Comment by Mehul1995** [comment]
The Search button onclick displays a toast notification saying "This is a sample app and the scope is to show the design pattern"

**Comment by Mehul1995** [comment]
Here's the video of the design in action - [Video](https://microsoft-my.sharepoint.com/personal/bimiron_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fbimiron%5Fmicrosoft%5Fcom%2FDocuments%2Fpersonal%2FAndroid%20Emulator%20%2D%20%5Fbuild%5F%5F5575%202020%2D11%2D17%2014%2D15%2D15%2Emp4&parent=%2Fpersonal%2Fbimiron%5Fmicrosoft%5Fcom%2FDocuments%2Fpersonal&or=teams&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOnY6L3AvYmltaXJvbi9FVkpZZHpqRGR6QkRqV1hGU3BTUkdid0J1dHhFcjhrbEY4RDlPYXo2TWMtZ2dnP3J0aW1lPXdBSkdHMDZMMkVn)

**Comment by Mehul1995** [comment]
[Here's another video in a dual landscape orientation](https://microsoft-my.sharepoint.com/:v:/p/bimiron/ETwOaRuWdrZCss21SVE0flABY0tke0cuI9rfA4uFVeAM-g)

**Comment by Mehul1995** [comment]
I've requested Bianca to also increase the time in the toast notification so that devs can read them without the message disappearing faster.

**Comment by CesarValiente** [comment]
I'm agree with @joyl1216 about the search button. If we don't provide any functionality and instead just we show a toast, we could get rid of that button, so there is no possible interaction with the button since, at the end, we don't provide any value.

**Comment by Mehul1995** [comment]
Sure! We can remove the search icon.

**Comment by bimiron** [comment]
Screenshots for the new items

Screenshot 2020-11-24 140930
Screenshot 2020-11-24 140948
Screenshot 2020-11-24 141036
Screenshot 2020-11-24 141101

**Review by Mehul1995 — APPROVED** [review]

**Review by joyl1216 — APPROVED** [review]

**Comment by joyl1216** [comment]
Looks awesome! @bimiron

File: DualView/src/androidTest/java/com/microsoft/device/display/samples/contentcontext/ParcelableTest.kt [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.microsoft.device.display.samples.contentcontext.model.Restaurant
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.hamcrest.CoreMatchers.`is` as iz
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+@SmallTest
+class ParcelableTest {
```
Comment (CesarValiente) [inline DualView/src/androidTest/java/com/microsoft/device/display/samples/contentcontext/ParcelableTest.kt:20]: Everything looks good! if anything to add is here, that I believe you are testing something that we expect the framework has already tested: the parcelable creation. If so, we could get rid of this test, but it's already done and here so it doesn't hurt. Your call Bianca ;-)

**Review by CesarValiente — APPROVED** [review]
Looks great Bianca! very good job! 👏👏👏

**Pull request merged**